src/js/components/help/components/pages/backstage/scenes/event-feedback/EventFeedback.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import styled from 'styled-components'
+import Sidebar from '../../../../help-topics'
+import Topic from '../../../../topic'
+import Desrciption from '../../../../help-description'
+import Description from '../../../../help-description';
```

Comment:
Can we import these like `js/components/help/components/help-description`? It'll be great if we avoid relative imports when the hierarchy is more than 3 levels apart.